crud.py

```diff
@@ -0,0 +1,70 @@
+from sqlalchemy.orm import Session
+
+import models
+import schemas
+
+
+def get_all_authors(
+        db: Session,
+        skip: int | None,
+        limit: int | None = None,
+) -> list[models.Author]:
+    query = db.query(models.Author)
+
+    if limit:
+        query = query.limit(limit)
+
+    if skip:
+        query = query.offset(skip)
+
+    return query.all()
+
+
+def create_author(db: Session, author: schemas.AuthorCreate) -> models.Author:
+    db_author = models.Author(
+        name=author.name,
+        bio=author.bio
+    )
+    db.add(db_author)
+    db.commit()
+    db.refresh(db_author)
+
+    return db_author
+
+
+def get_author_detail(db: Session, id: int) -> models.Author:
+    return db.query(models.Author).filter(models.Author.id == id).first()
```

Comment:
It's a good practice to handle the case when the author with the provided id does not exist in the database. Currently, the function will return None in this case, which might cause problems in the code that calls this function. Consider raising an exception if the author is not found.